roles/edpm_ovs/tasks/update.yml

```diff
@@ -0,0 +1,24 @@
+---
+- name: Update openvswitch packages without stopping the service
```

Comment:
is it possible to run just this role without edpm_update? (Thought that was the plan.)

Should there be molecule tests that target this role specifically?

Reply:
Please once you add molecule tests for the role, also enable them in `.github/workflows/molecule.yaml`.